### PR TITLE
Added github actions

### PR DESCRIPTION
**Pull request by amolopcito**

### CODE REVIEW

**Review by sabandi — APPROVED** [review]

File: .github/workflows/ansible-test.yml [hunk 1]
```diff
@@ -0,0 +1,153 @@
+name: Ansible CI
```
Comment (sabandi) [inline .github/workflows/ansible-test.yml:1]: rename the file to ci.yaml. Also is this file coming from `avi-dev`?

File: .github/workflows/ansible-test.yml [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+
+    - name: Set up Python 3.8
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.8
+
+    - name: Install ansible-base (v${{ matrix.ansible }})
+      run: pip install https://github.com/ansible/ansible/archive/v${{ matrix.ansible }}.tar.gz --disable-pip-version-check
+
+    - name: Build a collection tarball
```
Comment (sabandi) [inline .github/workflows/ansible-test.yml:24]: `Build a collection tarball` to `Build collection tarball`

File: .github/workflows/ansible-test.yml [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+      - name: Store galaxy_importer check log file
+        uses: actions/upload-artifact@v1
+        with:
+          name: galaxy-importer-log
+          path: .cache/collection-tarballs/importer_result.json
+
+  sanity:
+    name: Sanity in ubuntu-latest
+    needs:
+    - build
```
Comment (sabandi) [inline .github/workflows/ansible-test.yml:80]: shouldn't this be the other way around. sanity should be successful before we build.

File: .github/workflows/ansible-test.yml [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+
+    - name: Run sanity tests
+      run: ansible-test sanity --test ansible-doc --docker -v --color --truncate 0 --coverage --python 3.8
+      working-directory: /home/runner/.ansible/collections/ansible_collections/vmware/alb
+
+    - name: Run sanity tests
+      run: ansible-test sanity --test compile --docker -v --color --truncate 0 --coverage --python 3.8
+      working-directory: /home/runner/.ansible/collections/ansible_collections/vmware/alb
+
+  units:
```
Comment (sabandi) [inline .github/workflows/ansible-test.yml:122]: same as sanity.

File: .github/workflows/ansible-test.yml (hunk 1 above)
Comment (amolopcito) [reply .github/workflows/ansible-test.yml]: Done

**Review by sabandi — APPROVED** [review]

**Pull request merged**